src/components/cards/DirectoryCard.tsx

```diff
@@ -13,16 +14,12 @@ const DirectoryCard: CardComponent<
   return (
     <div className="Directorycard bg-white px-6 py-8 border h-full">
       <h3 className="mb-4 text-lg font-medium">
-        {profile.slug ? (
-          <Link
-            href={relativePrefixToRoot + profile.slug}
-            className="Link Link--primary hover:underline"
-          >
-            {profile.name}
-          </Link>
-        ) : (
-          profile.name
-        )}
+        <MaybeLink
+          className="Link Link--primary hover:underline"
```

Comment:
doesn't make a difference but there's also also `Link--underlineInverse` for this